src/js/services/meta.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+module.exports = function(app) {
+  app.factory(
+    'metaService',
+    [
+      function() {
+        var data = {
+          title: undefined,
+          author: undefined,
+          description: undefined,
+          keywords: undefined
+        };
```

Comment:
Ouch, calling `set` (if it's defined as above) would remove itself. ;)

We could do it this way:
```
function() {
  var service = {
    meta: {}
  };
  service.set = function(newData) {
    angular.copy(newData, service.meta);
  };
  return service;
}